### DISCUSSION TITLE
Remove selinux from dependencies

### DISCUSSION THREAD

**Discussion by sugymt** [ansible-community/molecule-plugins#27] [ec2]
The selinux module causes some problems if we don't use the system standard python.
Can selinux be removed from molecule-docker dependencies like molecule?

https://github.com/ansible-community/molecule/discussions/3424
https://github.com/ansible-community/molecule/pull/3388

**Comment by conn** [comment]
I've also ran into issues with this when running Molecule with pyenv on Rocky 8. Ansible and Molecule seem to run fine with Docker when I hand-patched `system_python` to use `sys.executable` instead of selecting a system Python. It also works just fine when I remove the selinux package after installing molecule-plugins.

There's also a fairly recent breaking change to how `system_python` is selected that upset some folks:
https://github.com/pycontribs/selinux/pull/56

It seems like this library doesn't offer anything but issues. I'd love to see it dropped.

**Comment by conn** [comment]
PR at #89
